docs/guides/Integration-Guides/tax-service-integration-guide/tax-services-specification.md

```diff
@@ -11,13 +11,8 @@ Below, we are going to go over tax service integration works, and if you want to
 
 In synchronous integration, VTEX’s Checkout API triggers and sends a request to the external tax service API whenever there are changes to a customer’s cart, such as adding or removing items.
 
-[block:callout]
-{
-  "type": "warning",
-  "body": "- Timeout for the request is five seconds.\n- There is no retry in case of timeout.\n- If the external service that responds to the request times out constantly, the store will not be able to finish the order.\n- If this integration is active, it applies to all stores in that account.",
-  "title": "When using Tax Service integration, keep in mind that:"
-}
-[/block]
+>⚠️ Timeout for the request is five seconds.\n- There is no retry in case of timeout.\n- If the external service that responds to the request times out constantly, the store will not be able to finish the order.\n- If this integration is active, it applies to all stores in that account.
```

Comment:
```suggestion
>⚠️ Timeout for the request is five seconds. There is no retry in case of timeout. If the external service that responds to the request times out constantly, the store will not be able to finish the order. If this integration is active, it applies to all stores in that account.
```